File: Daily Pratice Question/JAVASCRIPT/question(16).js
// 16. Check if all the elements in 
// arr = [3, 5, 9, 1, 7] are positive numbers, and print true or false.

let arr = [3, 5, 9, 1, 7] 
let flag=0;
for(i=0;i<arr.length;i++){
    if(arr[i]>0){
        flag++
    }
}if(flag==arr.length){
    console.log("true all elemnts are positive")
}else{
     console.log("false all elemnts are not positive")
}

File: Daily Pratice Question/C LANGUAGE/Question(16).c
// 16. Check if all the elements in arr = [3, 5, 9, 1, 7]
//  are positive numbers, and print true or false.


#include<stdio.h>
  int main(){

    int arr[] = {3, 5, -9, 1, 7};
    int i;
    int positive;
    int len=sizeof(arr)/sizeof(arr[0]);
    
    for(i=0;i<len;i++){
       if(arr[i]>0){
           positive++;
       }
    }
    if(len==positive){
        printf("all positive number");
    }
    else{
        printf("not all are positve numbers");
    }

return 0;
 }